src/style.css

```diff
@@ -1 +1,16 @@
 /* styles go here */
+.group {
+  margin: 20px auto;
+}
+
+input {
```

Comment:
use classname instead of tag selector

Reply:
I put a class and fixed formatting. Deployed successfully.
Could I know what the problem can be with my links?

Reply:
<img width="945" alt="Screenshot 2023-01-11 at 17 22 24" src="https://user-images.githubusercontent.com/53228259/211844962-4a10ab4b-5fd7-4dc8-a183-93394c4813c2.png">
change reame with your links

Reply:
DEMO LINK https://kseniaskr.github.io/layout_html-form/src/index.html - works
TEST LINK doesn't work
check instruction
https://mate-academy.github.io/layout_task-guideline/#how-to-solve-the-layout-tasks-on-github